homeassistant/components/vacuum/__init__.py

```diff
@@ -247,6 +247,8 @@ class StateVacuumEntity(
     _attr_supported_features: VacuumEntityFeature = VacuumEntityFeature(0)
```

Comment:
When deprecation is finally over, we would also remove VacuumEntityFeature.BATTERY.
Should we also test this and log to make sure there's no left-overs (as integrations might set it, but still not use either of the two props)?

Reply:
If we can do that, it's nice.